packages/server/shared/src/lib/system/system-prop.ts

```diff
@@ -95,6 +95,8 @@ export enum AppSystemProp {
   CODE_BLOCK_MEMORY_LIMIT_IN_MB = 'CODE_BLOCK_MEMORY_LIMIT_IN_MB',
 
   CANDU_CLIENT_TOKEN = 'CANDU_CLIENT_TOKEN',
+
+  EXECFILE_MAX_BUFFERSIZE_MB = 'EXECFILE_MAX_BUFFERSIZE_MB',
```

Comment:
The naming is a bit all over the place, `EXEC_FILE_MAX_BUFFER_SIZE_MB`

Reply:
Yea the german grammar was strong in that one

Reply:
Ah ireallylike germangrammer it's soeasytounderstandish